pkg/ginkgo-reporters/polarion_reporter.go

```diff
@@ -93,6 +93,7 @@ type PolarionReporter struct {
 	TestIDPrefix    string
 	TestRunTemplate string
 	TestRunTitle    string
+	ParallelNode    int
```

Comment:
I may miss  something, but where is this set?

Reply:
Here: https://github.com/kubevirt/kubevirt/pull/6908/files#diff-0a9a24f44dfc4029d6c2465ca6246bf66829e97d236da6ff80172a8a9c73866aR78

Reply:
I have marked the other PR as DNM, as it contains the changes in the vendor folder, which needs to be bumped properly when this PR goes in.